src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace System.Linq.Expressions.Interpreter
 {
+#if FEATURE_MAKE_RUN_METHODS
```

Comment:
Yes, the code guarded by those flags is exclusive to the features they represent. It doesn't hurt to double-check though (@VSadov). I've also created #4104 to investigate all of those flags.